### DISCUSSION TITLE
Debugging Pocketbase as a framework in VSCode

### DISCUSSION THREAD

**Discussion by blackfyre** [pocketbase/pocketbase#3635] [Q&A] [answered]
VScode support Go Delve as a debug tool, but I'm still green on that front (I've always relied on static check and gut feeling). Does anyone have a VSCode `launch.json` snippet willing to share? 😸

**Comment by user19820** [comment, marked as answer]
Late response, but for those who might look at this later, this works:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceRoot}",
            "args": ["serve"]
        }
    ]
}
```


If you want additional flags for your debugging, add them to "args".

**Reply by lordvcs** [reply]
this works, thanks

**Reply by eb-so** [reply]
Chad 💪
